specifications/api/swagger.yaml

```diff
@@ -1927,7 +1927,9 @@ paths:
         Some pools _may_ also have metadata attached to them.
       parameters:
         - *parametersWalletId
-      responses: *responsesListStakePools
+      responses:
+        - *responsesListStakePools
+        - *responsesErr404
```

Comment:
Instead, add `responsesErr404` to the possible responses of `responsesListStakePools`:

```yaml
x-responsesListStakePools: &responsesListStakePools
  <<: *responsesErr404
  <<: *responsesErr405
  200:
    description: Ok
    content:
      application/json:
        schema:
          oneOf:
            - type: array
              items: *ApiStakePool
              title: "cardano-node stake pools"
            - type: array
              items: *ApiJormungandrStakePool
              title: "jormungandr stake pools"
```